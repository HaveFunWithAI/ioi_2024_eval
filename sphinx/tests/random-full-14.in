250 2250
188 119 188 69 90 89 39 12 119 12 220 248 1 1 39 1 55 188 222 55 89 188 208 69 110 55 110 69 1 207 39 248 90 39 222 208 222 12 69 248 90 1 222 220 119 110 69 89 248 222 12 90 90 69 89 188 119 89 1 110 188 119 12 220 208 220 12 90 89 110 119 220 90 119 110 208 119 1 207 110 89 39 89 248 39 119 222 1 110 69 110 39 12 119 110 1 90 208 69 208 222 220 207 110 39 119 89 110 110 207 208 12 1 220 220 55 220 220 89 222 222 89 207 220 12 110 220 207 39 207 89 220 222 12 55 1 90 188 39 222 207 1 69 119 90 220 207 39 89 208 55 55 69 1 222 119 119 119 69 119 119 1 208 207 188 248 55 90 90 110 110 69 207 55 207 220 119 208 1 220 89 39 1 222 55 207 208 222 69 110 119 55 55 1 119 69 220 119 12 207 188 69 1 55 89 119 89 248 69 188 89 208 90 39 55 119 220 55 1 188 89 55 222 90 55 110 222 90 222 55 90 222 207 110 188 55 188 55 119 207 119 1 69 208 110 39 110 110 1 90
20 26
23 26
23 196
26 99
99 105
6 99
6 233
26 111
6 231
81 233
81 183
81 86
6 32
88 233
105 173
214 231
183 212
0 86
180 212
88 244
82 196
82 239
133 196
208 231
68 244
20 78
105 186
6 103
127 173
105 157
144 233
88 147
19 147
133 216
127 134
179 208
6 220
54 86
140 233
78 100
0 95
147 226
187 214
144 234
86 130
111 230
86 112
147 199
6 131
88 191
172 216
57 172
54 113
149 212
10 234
133 168
141 157
15 196
78 116
158 168
107 186
99 114
3 127
1 105
106 226
106 223
107 246
22 100
96 113
67 68
97 212
81 104
135 226
54 139
104 125
57 162
30 96
49 104
19 170
163 179
10 52
207 233
55 130
107 182
113 229
55 205
41 180
68 115
22 101
17 103
41 176
71 205
71 215
76 182
159 179
54 166
35 176
136 230
3 153
39 180
29 39
31 196
21 166
2 180
87 97
115 195
22 222
82 129
54 118
60 111
104 138
36 207
30 38
0 221
186 188
107 210
163 241
51 230
98 116
14 157
55 181
86 232
143 223
120 131
20 42
99 194
38 109
129 167
182 249
45 208
93 140
74 153
62 157
135 145
31 44
149 240
185 241
39 80
92 240
51 94
94 164
121 130
70 191
1 213
183 193
199 218
72 207
30 84
58 181
6 9
93 169
167 238
69 240
65 94
142 207
103 146
152 193
137 246
66 106
2 24
83 152
129 165
84 203
50 180
49 124
39 236
36 204
73 188
74 79
90 229
149 189
113 117
11 176
77 163
25 30
9 56
18 179
69 206
113 245
102 165
7 234
128 222
8 221
188 243
47 56
98 161
171 210
89 213
61 99
156 180
151 187
148 245
75 141
58 201
119 125
49 217
157 202
68 235
168 227
48 75
143 174
127 192
28 145
62 160
47 155
36 110
43 139
126 213
2 190
27 41
175 206
53 240
63 220
33 121
105 211
161 209
85 101
23 197
103 200
228 240
67 248
40 169
22 123
154 216
139 150
46 143
1 12
126 225
100 177
86 219
34 90
11 37
13 41
64 173
122 243
104 237
16 99
4 31
224 243
91 107
0 198
226 247
93 184
108 143
59 140
82 132
93 242
82 178
5 105
86 190
30 57
56 75
145 177
7 70
105 151
102 113
169 202
50 69
80 191
21 167
17 172
29 240
128 231
174 213
126 146
107 204
102 240
120 245
67 243
196 230
37 48
110 111
24 84
100 172
193 200
153 159
64 136
77 93
51 182
104 216
0 219
0 194
30 121
13 232
7 131
40 176
216 233
144 177
7 212
60 99
29 149
50 93
150 203
114 239
51 122
113 156
232 233
191 236
68 101
17 74
113 155
72 219
4 138
177 225
57 182
20 179
86 169
56 63
40 136
115 159
45 83
31 52
41 64
59 126
146 228
75 102
11 17
59 137
173 220
83 89
98 188
65 110
85 241
145 149
54 134
97 166
57 98
97 140
77 100
17 62
18 20
47 203
13 98
91 199
38 78
31 227
20 60
35 218
3 35
126 206
196 204
13 31
177 178
114 158
3 103
184 201
54 60
150 238
64 163
10 169
196 244
51 57
2 58
20 145
5 124
70 80
178 226
113 241
113 186
24 42
12 38
102 118
39 46
108 239
139 169
1 189
114 137
17 44
46 247
115 228
103 241
93 119
131 232
9 110
108 231
122 186
161 212
112 205
51 222
144 198
132 207
165 189
41 68
33 39
68 141
8 203
157 240
179 230
5 55
156 241
208 226
166 200
105 171
110 119
22 234
119 240
45 160
33 235
110 179
82 199
95 106
13 201
106 241
39 187
139 192
134 186
81 111
8 100
123 186
19 118
99 190
59 214
67 126
25 207
187 243
65 88
76 239
52 89
133 135
144 202
132 196
205 213
32 210
151 175
92 168
47 143
16 26
183 217
191 221
145 157
178 246
188 189
36 106
11 39
3 50
123 203
157 183
117 164
108 135
130 227
60 181
30 219
49 238
106 120
100 139
12 45
74 193
48 111
111 220
33 204
23 231
4 181
12 235
172 224
110 161
31 107
9 98
35 226
50 245
40 222
182 239
66 92
115 132
168 219
62 198
20 67
53 220
41 153
114 200
58 144
83 208
92 140
29 173
1 178
21 127
113 140
157 198
50 212
93 171
74 155
64 83
22 189
25 54
24 46
84 112
87 104
42 248
78 170
110 187
72 214
56 203
143 185
70 129
4 192
172 191
220 234
133 207
26 198
15 140
0 11
125 243
128 246
49 181
77 117
107 201
5 91
2 234
97 125
104 110
32 176
52 102
30 106
130 150
184 198
33 236
16 126
61 144
17 84
50 77
98 200
26 38
43 149
30 52
61 179
34 241
16 18
146 218
40 178
61 195
171 237
77 159
16 48
99 172
78 205
28 161
25 150
154 218
7 71
119 197
115 151
15 155
34 42
105 162
26 121
105 143
80 134
57 99
8 137
155 169
147 187
110 160
165 219
41 72
196 214
68 73
194 216
62 201
15 84
86 126
54 171
172 180
67 178
36 214
42 87
103 141
171 191
17 170
113 127
119 225
230 236
24 179
105 200
77 195
27 81
129 199
139 179
42 154
15 81
13 188
77 228
16 247
139 171
185 221
60 229
177 192
103 144
116 186
98 199
95 152
31 94
120 243
57 72
16 91
8 147
153 238
51 103
46 192
68 85
216 241
81 187
9 230
85 152
86 122
78 167
56 229
58 246
134 145
112 200
92 113
21 51
134 178
21 168
222 244
89 246
116 187
139 154
4 194
200 223
102 159
44 56
0 67
30 225
67 247
20 180
34 56
123 193
24 149
140 141
12 120
159 225
134 202
139 203
86 120
146 169
60 87
35 42
128 219
132 240
92 186
1 198
133 232
6 202
156 236
152 219
7 238
5 164
137 147
153 172
39 41
55 239
20 83
53 150
198 206
1 174
110 195
89 193
79 121
216 231
173 244
29 212
173 180
96 239
7 196
125 133
91 190
29 87
203 248
5 27
89 173
80 100
110 245
28 39
58 198
108 184
31 86
58 62
10 63
79 190
109 164
86 223
30 139
147 174
36 63
10 155
143 211
43 158
211 214
232 244
42 231
40 111
75 84
54 224
11 181
17 96
203 227
73 191
69 147
181 228
16 64
76 246
111 242
153 225
1 51
35 123
131 237
31 87
0 238
72 135
133 229
67 217
39 138
195 228
93 243
88 141
109 151
9 146
39 69
23 162
28 179
93 132
3 13
85 172
78 228
29 154
155 176
15 27
227 235
2 98
78 133
7 247
126 130
2 158
28 159
11 42
44 215
40 146
89 135
1 163
188 214
92 117
18 85
127 133
85 195
43 202
84 110
63 70
16 80
98 101
42 44
70 89
17 151
79 220
161 217
19 109
197 240
159 184
41 165
64 232
83 155
17 194
107 215
104 226
7 214
22 117
15 120
58 187
73 164
58 208
4 43
163 239
98 165
76 101
7 91
1 231
168 207
14 169
120 234
154 205
160 162
68 77
88 115
128 189
25 210
117 131
25 156
87 172
176 206
157 218
62 177
209 220
15 156
93 112
54 230
81 229
196 205
78 161
74 208
126 238
104 144
129 166
36 236
69 246
53 64
51 226
61 104
72 192
92 161
36 188
180 203
72 108
18 60
64 233
63 225
4 55
35 144
161 226
25 52
51 225
77 113
101 159
27 231
170 235
148 235
94 153
126 182
123 127
64 124
0 23
24 162
127 164
30 144
74 135
93 183
208 209
28 116
105 182
98 172
111 219
104 175
135 162
20 110
215 245
23 50
114 170
35 101
24 229
55 132
77 246
128 199
6 189
66 179
35 143
60 201
15 149
156 247
87 215
38 134
48 147
32 75
15 58
80 184
62 208
14 66
70 173
131 138
10 196
48 141
23 54
196 232
153 220
199 241
32 212
129 154
7 104
39 74
18 123
207 220
62 194
49 85
22 184
146 185
52 78
1 152
219 228
90 248
17 244
22 86
6 41
95 156
78 136
83 105
88 123
57 125
15 200
17 121
32 77
54 222
152 183
55 198
20 37
177 227
54 214
50 97
199 226
151 156
129 159
179 196
40 203
27 99
83 210
55 155
139 216
47 205
233 241
158 216
41 187
98 248
4 198
34 35
39 72
41 166
32 238
36 115
93 109
200 227
26 124
114 136
60 92
193 198
30 215
65 109
52 189
53 201
7 79
59 159
75 187
159 235
22 227
71 139
23 203
216 247
42 191
33 97
141 199
75 169
16 76
44 59
25 132
130 152
22 196
91 160
30 73
99 145
17 164
31 61
100 190
12 73
52 126
178 206
29 181
171 234
43 177
112 231
33 43
50 58
132 170
106 190
93 95
14 49
127 188
89 194
122 185
38 214
53 124
185 249
72 81
59 106
187 208
158 176
141 179
3 99
18 205
192 230
23 24
33 77
70 228
2 129
125 134
42 206
41 126
154 217
115 131
86 156
66 235
92 246
164 173
43 98
10 173
19 247
151 202
59 108
60 171
46 135
201 235
67 181
183 215
31 190
169 192
48 130
149 215
134 163
29 211
82 166
134 249
97 187
80 89
124 212
85 153
165 174
185 195
95 180
144 193
115 130
170 196
126 153
13 136
13 200
144 153
11 148
98 120
38 165
103 129
40 114
37 243
23 87
139 234
34 94
180 231
39 132
106 208
38 74
71 81
134 157
48 191
34 71
126 215
71 238
75 150
117 218
45 188
138 182
179 216
35 195
22 232
21 142
57 87
100 205
14 125
125 177
203 241
26 61
226 231
75 101
151 199
155 210
70 226
6 36
95 203
122 248
7 158
172 244
98 184
75 178
29 218
138 156
40 100
19 85
0 237
85 188
111 117
70 71
138 217
132 159
222 228
54 156
59 110
0 7
161 246
45 243
63 233
207 226
170 180
43 192
132 152
65 96
192 208
88 129
185 243
147 246
15 71
137 168
206 213
62 227
182 185
161 163
79 225
74 214
58 218
135 169
91 185
179 227
9 189
116 139
3 123
87 156
28 202
207 216
23 91
50 57
8 125
116 173
42 176
80 102
113 212
101 180
97 185
104 113
99 154
47 67
148 180
221 242
41 95
58 100
28 225
163 245
173 226
200 225
2 227
0 120
15 129
8 215
97 225
33 69
153 166
137 218
97 122
233 249
88 249
47 127
60 239
62 97
46 102
204 231
193 210
108 227
104 165
33 226
46 216
205 220
30 197
114 222
62 123
174 240
27 173
25 94
39 249
94 200
209 216
37 129
162 176
28 118
183 187
128 239
66 248
2 116
133 242
87 231
85 169
39 90
33 48
81 232
41 244
0 21
136 137
84 222
7 44
83 87
174 233
42 149
29 120
189 246
44 108
150 153
108 182
29 69
71 240
51 189
84 171
115 202
189 247
115 230
64 194
18 63
163 203
30 135
13 209
48 216
33 153
192 198
140 176
168 212
223 226
37 98
48 178
188 220
44 169
192 194
12 39
150 154
122 174
94 219
32 188
50 113
29 155
136 186
153 223
68 163
139 190
44 212
201 216
136 154
177 180
107 184
19 105
21 180
7 147
59 80
35 238
62 161
16 62
17 178
78 230
152 154
133 231
79 232
91 224
4 139
8 122
135 183
59 205
202 233
115 246
70 217
32 71
68 249
154 246
99 236
89 221
14 146
65 139
114 123
175 196
92 183
161 221
33 202
70 175
44 120
10 93
13 211
44 119
42 113
139 222
85 203
72 200
96 122
117 156
75 219
91 105
140 220
59 70
177 200
1 140
75 237
137 244
138 223
35 135
26 135
42 82
23 112
76 115
238 244
81 218
177 238
172 205
51 202
198 212
43 212
102 233
150 247
81 154
5 74
167 243
75 185
209 221
119 143
32 174
25 193
223 224
91 219
81 126
9 234
102 216
13 55
139 219
103 173
84 127
108 194
44 55
14 34
0 36
204 215
138 154
93 180
117 122
0 101
17 109
11 207
70 219
41 80
15 117
20 181
56 213
32 64
165 241
20 74
70 88
71 138
116 162
118 149
120 200
40 79
8 101
18 193
146 176
52 70
48 214
51 199
199 235
137 228
66 70
103 233
57 220
100 232
75 85
176 229
10 28
130 242
17 49
129 189
67 168
2 177
141 222
17 192
119 180
153 156
166 207
34 205
19 245
159 218
148 204
188 246
18 155
114 243
200 239
39 93
3 202
75 82
182 210
219 229
1 120
124 134
112 149
21 238
13 43
37 218
156 194
148 149
17 223
146 205
13 70
51 93
0 48
94 226
165 227
28 190
50 82
11 26
20 58
36 85
6 46
119 209
70 199
21 218
36 38
128 226
174 242
81 240
16 194
34 181
53 155
68 247
135 151
188 219
76 167
140 156
144 241
175 179
104 208
89 158
100 204
131 208
113 243
174 215
87 246
74 131
64 96
225 237
42 170
83 159
73 230
40 182
30 79
113 149
41 119
109 176
23 108
7 174
23 241
40 235
46 100
45 58
125 217
63 160
133 158
113 125
0 218
77 94
107 217
85 134
100 144
54 226
192 226
213 218
65 163
60 241
207 247
9 162
45 219
48 95
106 138
39 221
38 172
50 54
44 118
159 187
45 73
191 226
25 162
112 219
215 225
100 109
88 104
101 247
16 108
23 135
21 53
68 157
88 155
2 70
91 211
11 245
6 116
81 158
17 57
55 107
192 240
135 138
224 240
44 213
142 179
80 162
241 249
106 128
112 249
161 249
211 233
84 133
173 215
202 232
241 243
72 141
128 163
61 71
26 28
195 219
26 157
86 153
153 160
134 193
11 122
85 197
3 120
129 231
74 215
43 214
10 106
48 185
177 230
81 137
69 164
98 160
52 129
219 235
85 141
110 247
139 181
64 140
32 63
180 221
123 210
41 145
61 123
50 199
77 249
107 183
39 237
197 207
75 89
125 153
164 186
132 146
69 80
53 179
35 243
74 228
90 169
83 197
86 178
76 106
124 175
55 229
48 127
128 148
27 171
2 26
18 139
17 86
101 133
58 69
91 230
89 155
51 229
18 135
8 162
79 89
141 229
102 123
60 147
196 218
16 195
13 20
170 199
80 168
40 164
129 198
106 217
68 224
77 174
39 127
206 212
112 135
171 186
153 194
185 220
96 115
2 93
165 245
79 240
54 75
107 133
50 230
105 199
33 60
64 71
34 154
48 60
140 186
47 153
32 84
36 103
69 75
102 248
0 61
35 83
92 172
21 71
200 214
64 157
30 239
146 153
46 189
76 226
98 219
191 212
77 96
40 193
223 230
14 88
159 204
76 173
165 233
29 150
167 241
40 195
13 33
2 41
12 242
138 225
109 212
129 135
51 63
215 244
22 81
13 95
76 84
47 156
47 171
44 135
105 194
30 115
22 194
146 196
134 191
61 222
53 126
85 239
76 127
164 177
49 220
119 155
7 193
55 199
115 212
177 209
155 232
46 138
45 198
19 178
1 182
12 43
8 189
13 127
41 134
78 178
104 182
88 181
51 154
45 52
193 216
65 193
123 241
42 169
38 71
127 158
68 195
129 249
40 128
82 171
57 86
125 141
167 218
34 195
13 29
162 199
60 139
157 248
84 240
72 140
25 119
157 221
162 230
59 229
19 72
61 169
70 117
32 93
16 201
203 237
23 221
117 119
221 236
173 184
23 115
104 183
128 201
120 154
21 247
22 99
126 226
86 88
10 65
36 215
4 189
71 184
6 61
125 155
20 111
47 147
25 246
174 179
69 173
2 40
45 92
31 235
13 228
41 169
144 157
114 177
131 200
135 238
24 154
218 225
48 96
115 161
121 159
186 218
6 45
165 176
35 155
10 193
119 202
66 230
187 223
56 68
129 169
48 116
99 133
79 98
206 226
132 149
43 53
53 243
33 184
44 101
34 110
3 43
4 37
43 109
6 93
70 132
1 129
9 141
65 155
83 145
30 188
13 89
127 163
89 101
15 194
108 247
113 189
62 207
15 232
91 215
84 248
3 217
7 159
104 131
112 218
60 137
74 172
82 234
151 154
225 235
20 193
32 148
31 115
201 248
56 139
128 221
121 175
32 54
140 159
66 181
0 145
18 159
29 113
201 211
13 169
97 197
16 92
124 147
123 133
62 203
38 49
157 172
37 245
107 135
56 215
168 182
11 220
187 207
46 241
110 244
63 197
59 224
32 216
51 249
113 121
104 174
224 236
104 162
124 171
30 128
44 239
13 19
23 218
17 94
108 152
76 138
209 240
108 207
146 168
4 167
104 108
91 192
67 185
99 112
87 186
181 196
116 206
68 198
139 208
12 208
183 236
94 139
37 137
109 168
189 191
144 209
177 234
101 190
15 100
23 144
30 192
72 122
57 214
40 112
89 104
15 206
159 186
68 213
189 214
184 220
11 40
59 157
115 210
80 171
21 239
16 211
78 198
94 122
146 160
4 219
0 80
104 196
33 211
55 106
36 224
28 90
201 213
129 222
96 195
47 115
36 191
74 121
7 135
11 182
59 194
14 101
99 241
82 180
18 153
81 226
21 173
47 131
97 211
90 181
141 143
142 225
5 47
49 82
1 10
126 246
32 46
138 146
181 240
55 123
188 238
55 243
64 169
118 213
0 214
174 241
1 29
34 46
174 249
68 84
11 195
6 132
159 247
118 186
32 141
82 243
88 111
44 132
28 215
87 92
179 248
32 33
18 47
126 239
82 216
17 147
106 171
70 223
5 38
107 142
35 78
49 54
95 139
85 193
61 177
71 241
0 229
16 47
203 228
42 233
59 182
176 211
130 133
72 238
131 188
13 115
110 132
60 231
53 135
98 204
91 95
157 182
34 91
49 122
111 138
165 181
114 214
89 199
121 142
61 90
156 208
55 84
88 146
22 30
132 177
13 47
15 109
102 243
98 135
74 119
54 204
14 244
50 162
155 236
1 11
137 153
74 108
9 174
156 165
15 184
7 165
86 246
99 110
10 213
139 147
141 190
230 247
150 163
115 133
41 247
116 188
102 186
101 246
10 246
192 248
137 205
62 217
186 246
127 242
13 128
216 228
73 249
55 224
88 95
119 230
115 139
2 220
19 69
144 185
32 192
5 236
23 100
66 134
140 148
5 229
111 170
70 195
168 228
119 181
64 126
77 84
142 232
22 233
23 99
134 240
61 77
67 230
99 202
56 57
160 230
23 139
30 160
120 210
136 173
6 242
85 159
73 232
10 224
79 177
72 95
112 202
93 202
17 217
137 175
88 221
50 124
33 81
153 178
115 174
34 176
0 33
88 167
33 189
114 242
218 246
160 209
28 49
160 180
122 132
22 40
88 108
43 188
59 122
15 111
21 190
137 231
37 127
0 200
110 131
28 110
145 206
37 41
65 224
25 56
49 89
30 169
3 205
6 10
65 121
39 87
115 128
4 111
68 166
85 182
38 249
73 222
130 148
104 140
20 218
131 167
30 50
102 219
83 207
117 180
160 170
22 34
33 42
7 19
119 169
70 114
3 223
152 189
53 110
29 33
46 181
62 149
145 147
111 172
165 180
100 145
110 185
75 136
26 67
161 202
42 108
194 204
71 232
34 87
21 81
52 199
90 184
153 227
132 161
46 198
102 224
22 63
64 228
82 211
55 201
86 221
86 239
67 110
81 152
187 199
49 92
220 229
41 248
176 185
52 229
23 59
8 198
20 240
119 224
74 94
39 183
50 110
126 137
117 166
122 231
14 200
4 101
162 217
80 249
103 159
34 72
43 99
91 173
47 110
49 91
68 125
4 174
191 194
105 158
129 209
73 187
111 192
29 164
144 223
40 144
160 232
49 212
43 55
111 173
224 241
168 169
62 190
34 55
25 238
67 233
92 167
51 165
21 56
68 227
86 116
156 207
99 139
151 163
169 193
34 138
15 118
161 180
60 215
19 61
95 123
49 127
181 214
24 74
167 210
96 172
198 232
87 198
52 248
108 190
143 155
152 187
172 173
62 67
107 163
202 239
16 230
2 119
95 126
203 247
105 203
153 246
9 233
31 133
18 145
110 198
172 195
50 159
143 169
155 181
149 169
93 97
30 61
74 232
75 108
77 126
0 103
78 165
25 163
147 202
186 240
40 50
113 200
36 168
47 50
105 220
71 142
142 150
53 162
