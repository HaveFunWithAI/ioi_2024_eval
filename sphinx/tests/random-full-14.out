250 2348
188 119 188 69 90 89 39 12 119 12 220 248 1 1 39 1 55 188 222 55 89 188 208 69 110 55 110 69 1 207 39 248 90 39 222 208 222 12 69 248 90 1 222 220 119 110 69 89 248 222 12 90 90 69 89 188 119 89 1 110 188 119 12 220 208 220 12 90 89 110 119 220 90 119 110 208 119 1 207 110 89 39 89 248 39 119 222 1 110 69 110 39 12 119 110 1 90 208 69 208 222 220 207 110 39 119 89 110 110 207 208 12 1 220 220 55 220 220 89 222 222 89 207 220 12 110 220 207 39 207 89 220 222 12 55 1 90 188 39 222 207 1 69 119 90 220 207 39 89 208 55 55 69 1 222 119 119 119 69 119 119 1 208 207 188 248 55 90 90 110 110 69 207 55 207 220 119 208 1 220 89 39 1 222 55 207 208 222 69 110 119 55 55 1 119 69 220 119 12 207 188 69 1 55 89 119 89 248 69 188 89 208 90 39 55 119 220 55 1 188 89 55 222 90 55 110 222 90 222 55 90 222 207 110 188 55 188 55 119 207 119 1 69 208 110 39 110 110 1 90
